client/lib/post-normalizer/rule-content-make-images-safe.js

```diff
@@ -60,7 +60,7 @@ function isCandidateForContentImage( imageUrl ) {
 	} );
 }
 
-export default function( maxWidth ) {
```

Comment:
i mean, the wrapped function is still using it, but callers no longer have to use it. Notice the caller is still invoking the function with `()`